pkg/credentials/provider_test.go

```diff
@@ -0,0 +1,57 @@
+package credentials
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go/service/sts"
```

Comment:
nit: ordering

Reply:
Aha: goimports is a thing. I'll configure GoLand to handle that automatically and also add it to our Github Actions.